src/lib/components/composites/settings/project-settings/slr/FetcherAddDialog.svelte

```diff
@@ -0,0 +1,102 @@
+<script lang="ts">
+    import AlertDialog from "$lib/components/composites/dialog/AlertDialog.svelte";
+    import Alert from "$lib/components/composites/utils/Alert.svelte";
+    import * as Select from "$lib/components/primitives/select";
+    import { Project, Project_Settings } from "$lib/model/api/project";
+    import type { ApiError } from "$lib/model/general";
+    import { updateFetchers } from "./UpdateFetchers";
+
+    interface Props {
+        projectId: string;
+        projectSettings: Project_Settings | undefined;
+        unusedFetchers: string[];
+        onProjectChanged: (project: Project) => void;
+        open: boolean;
+    }
+
+    let {
+        projectId,
+        projectSettings,
+        onProjectChanged,
+        unusedFetchers,
+        open = $bindable(),
+    }: Props = $props();
+
+    let error: ApiError | undefined = $state();
+    let loading = $state(false);
+    let fetchers: string[] = $state([]);
+    const content = $derived(
+        fetchers.length === 0
+            ? "Select a fetcher"
+            : `${fetchers.length} fetcher${fetchers.length > 0 ? "s" : ""} selected`,
```

Comment:
This doesn't work. Also, we already have an extra method for this.
```suggestion
            : `${fetchers.length} ${pluralize(fetchers.length, "fetcher", "fetchers")} selected`,
```

Reply:
Oops. Off by one, should have been a `1`. Anyway, the pluralize function is now used.